Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -6623,7 +6630,16 @@ public void HandleActUIElement(ActUIElement act)
                         MoveToElementActions(act);
                         break;
                     case ActUIElement.eElementAction.IsEnabled:
-                        act.AddOrUpdateReturnParamActual("Enabled", e.Enabled.ToString());
+                        if (e == null)
```

Comment:
Not required for isEnabled. E.g. if the element is not present then returning false is wrong